.vitepress/theme/components/CustomLayout.vue

```diff
@@ -7,7 +7,9 @@ const { Layout } = DefaultTheme;
 <template>
   <Layout>
     <template #doc-footer-before>
-      <Feedback />
+      <ClientOnly>
+        <Feedback />
+      </ClientOnly>
```

Comment:
My understanding is that the `<ClientOnly>` component is globally registered and does not need to be imported. (It's not exported from `'vitepress'` either.) Seems to work in my tests. I wish the VitePress docs were clearer about that.